healthcheck.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 PORT=${FB_PORT:-$(jq .port /.filebrowser.json)}
```

Comment:
Would it make any sense to also add the `-r` flag to the port query, just incase the user defined the port in the config file as a string, for example:

```json
{
  "port": "80",
  "baseURL": "",
  "address": "",
  "log": "stdout",
  "database": "/database.db",
  "root": "/srv"
}
```

```shell
# jq .port filebrowser.json
"80"
# jq -r .port filebrowser.json
80
```

Reply:
I see no harm in doing that. I'll commit that now.